Automattic/wp-calypso
# BlazePress Posts tab redesign (mobile version)

Related to [SELFSERVE-505](https://jira.tumblr.net/browse/SELFSERVE-505)

Designs in [Figma](https://www.figma.com/file/uvh2KtY65b7hGBHI9exbJ1/BlazePress?type=design&node-id=6461-60371&t=fp0excRoGIeDL2y1-0)

## Proposed Changes
- Adjust styles according to designs,
- Switch to using data from the new API endpoint created by @jjolmo,

## Not in this PR:
In Figma placeholder contains the sign "Sort:" but I have not found the way how to implement it, probably in the next iteration:
<img width="904" alt="Screenshot 2023-05-25 at 20 39 01" src="https://github.com/Automattic/wp-calypso/assets/115007291/5bcfcc9a-8c03-4764-b832-62eb68858aaa">

## Testing Instructions



- Run Calypso locally by calling `yarn && yarn start`
- Since the page we will test uses the new endpoint for posts we need to make sure it contains required fixed (provided in the diff D111879-code), so if the diff is not merged, please use your sandbox with that diff instead of WPCOM public API
- Use the following URL to open a page with posts tab: http://calypso.localhost:3000/advertising/timurads.wordpress.com/posts
- Make sure desktop view looks good:
![blaze-press-desktop](https://github.com/Automattic/wp-calypso/assets/115007291/9c19b53a-4e11-44bd-afab-a67e18f7e17d)

- Then use Mobile view from your browser's dev tools, the mobile version should look like in Figma:
<img width="922" alt="Screenshot 2023-05-25 at 20 30 44" src="https://github.com/Automattic/wp-calypso/assets/115007291/64b6c896-2ae5-43a9-8e8a-aece2f546ea4">


## Pre-merge Checklist
